doc/getting-started/rt_industry_ubuntu.rst

```diff
@@ -321,6 +317,37 @@ following steps:
    $ make clean && make iasl
    $ sudo cp ./generate/unix/bin/iasl /usr/sbin/
 
+Build and Install the RT kernel for the Ubuntu User VM
+------------------------------------------------------
+
+Follow these instructions to build the RT kernel.
+
+#. Clone the RT kernel source code:
+
+   .. code-block:: none
+
+      $ git clone https://github.com/projectacrn/acrn-kernel/tree/4.19/preempt-rt
+
+#. Build the kernel:
+
+   .. code-block:: none
+
+      $make olddefconfig
```

Comment:
We also need to get a config file from somewhere and copy it as `.config` before running this command. @fuzhongl , which should be used? Or is the default (i.e. `arch/x86/configs/x86_64_defconfig`) suitable for this?

Reply:
@gvancuts I am going to merge this PR even though we have not yet received a response from @fuzhongl. We can submit a new PR when we receive this info.